<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>1031</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustIgnored</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <widget class="QWidget" name="img">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>665</width>
         <height>932</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionopen_file"/>
    <addaction name="actionopen_dir"/>
    <addaction name="actionLoad_File"/>
    <addaction name="actionLoad_Dir"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuEditr">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionReDo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEditr"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="img_action"/>
   <addaction name="file_action"/>
   <addaction name="save_action"/>
   <addaction name="saveAs_action"/>
   <addaction name="unDo_action"/>
   <addaction name="reDo_action"/>
   <addaction name="pre_action"/>
   <addaction name="next_action"/>
   <addaction name="scanOut_action"/>
   <addaction name="scanIn_action"/>
   <addaction name="init_action"/>
  </widget>
  <widget class="QDockWidget" name="annoDocker">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Annotations</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomListWidget" name="annoList">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="labelDocker">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Labels</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CustomListWidget" name="labelsList"/>
     </item>
     <item>
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QTextEdit" name="labelEdit">
          <property name="maximumSize">
           <size>
            <width>226</width>
            <height>33</height>
           </size>
          </property>
          <property name="lineWrapMode">
           <enum>QTextEdit::NoWrap</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdd">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>51</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>add</string>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="filesDocker">
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Files</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="filesList"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionopen_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/file.png</normaloff>:/icons/file.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionopen_dir">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open dir</string>
   </property>
  </action>
  <action name="actionLoad_File">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/copy.png</normaloff>:/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Load File</string>
   </property>
  </action>
  <action name="actionLoad_Dir">
   <property name="text">
    <string>Load Dir</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionReDo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="img_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/expert.png</normaloff>:/icons/expert.png</iconset>
   </property>
   <property name="text">
    <string>打开图片</string>
   </property>
   <property name="toolTip">
    <string>打开新图片</string>
   </property>
  </action>
  <action name="file_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>打开文件夹</string>
   </property>
   <property name="toolTip">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="save_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="saveAs_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save-as.png</normaloff>:/icons/save-as.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>另存为</string>
   </property>
  </action>
  <action name="unDo_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/undo.png</normaloff>:/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
  </action>
  <action name="reDo_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/redo.png</normaloff>:/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
   <property name="toolTip">
    <string>下一步</string>
   </property>
  </action>
  <action name="pre_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
   </property>
   <property name="text">
    <string>上一张</string>
   </property>
   <property name="toolTip">
    <string>上一张</string>
   </property>
  </action>
  <action name="next_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
   </property>
   <property name="text">
    <string>下一张</string>
   </property>
   <property name="toolTip">
    <string>下一张</string>
   </property>
  </action>
  <action name="scanOut_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/zoom-in.png</normaloff>:/icons/zoom-in.png</iconset>
   </property>
   <property name="text">
    <string>放大</string>
   </property>
   <property name="toolTip">
    <string>放大</string>
   </property>
  </action>
  <action name="scanIn_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/zoom-out.png</normaloff>:/icons/zoom-out.png</iconset>
   </property>
   <property name="text">
    <string>缩小</string>
   </property>
   <property name="toolTip">
    <string>缩小</string>
   </property>
  </action>
  <action name="init_action">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/zoom.png</normaloff>:/icons/zoom.png</iconset>
   </property>
   <property name="text">
    <string>复原</string>
   </property>
   <property name="toolTip">
    <string>复原</string>
   </property>
  </action>
  <zorder>annoDocker</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">widgets/customListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
